docs/UserGuide.md

```diff
@@ -3,112 +3,162 @@ layout: page
+typing but is unfamiliar with **Command Line Interface (CLI)** and other technical details. 
+
+## Purpose of this Guide
+
+The primary purpose of the user guide is to quickly help novice users to familiarize with the keyboard-based environment 
+and learn how to use the basic function of the app. The secondary purpose is to provide a fast way for the experienced users
+to find the app information that they are looking for.
+
+## Navigating the guide
+
+For quickly finding the information that you are looking, you can make use of the [Table of Contents](#table-of-contents) 
+below, which has a list of all the information, by clicking on the items, you will be directly to the content that you just clicked.
+
+Throughout the guide, there are also some icons which represents information outside just technicalities.
+
+* :bulb: **Tip**: This icon indicates a tip, suggestion, or piece of advice to help you use OH more effectively.
+
+* :information_source: **Note**: This icon tells you about some key details of the app usage that you could miss.
+
+* :warning: **Warning**: This icon alerts you about potential issues that you may face when using the app.
+
+* :star: **Benefit**: This icon signifies the benefits and relevance of an OH feature.
+
+When reading through the guide, you can look for these icons for extra information.
+
+*The usage of these special icons is inspired by section 1.4 of the user guide of the app SweetRewards [link](https://ay2324s2-cs2103t-t13-4.github.io/tp/UserGuide.html#14-navigating-this-guide)*
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+# Table of Contents
+  {:toc}
+
+
+# 1. Getting started
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
 2. Download the latest `OfficeHarbor.jar` from
    [here](https://github.com/AY2324S2-CS2103T-W13-2/tp/releases/latest).
 
-3. Copy the file to the folder you want to use as the _home folder_ for your OH.
+3. Copy the file to the folder you want to store and use your OH.
 
-4. (a) Double-click on the `officeharbor.jar` file to run the application.
-   (b) (Or) Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar officeharbor.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Double-click on the `officeharbor.jar` file to run the application.
+   An app similar to the below image should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing **| Enter &#9166; |** will open the help window.<br>
+5. Type the command in the command box and press Enter to run it. e.g. typing **`help`** and pressing **| Enter &#9166; |** will open the help window.<br>
```

Comment:
Now that I look at this, what the command box is is not really that obvious. Maybe we can annotate the previous Ui image?

(I think it's the same as the front page one, so we'd need to duplicate it)